exercises/go-1-b/.meta/example_struct.go

```diff
@@ -0,0 +1,83 @@
+package go1b
+
+const maxScore = 21
+
+type Card int
+
+const (
+	Ace   Card = 11
+	King       = 10
+	Queen      = 10
+	Jack       = 10
+	Ten        = 10
+	Nine       = 9
+	Eight      = 8
+	Seven      = 7
+	Six        = 6
+	Five       = 5
+	Four       = 4
+	Three      = 3
+	Two        = 2
+)
+
+type Hand struct {
```

Comment:
Wondering why this is exported. Everything else that would make this valueable to be exported is not exported: `score` method and `parseHand`.